src/components/NFTOriginVerifier.tsx

```diff
@@ -206,15 +272,20 @@ export default function NFTOriginVerifier() {
     lookupChain,
     lookupAsset,
     lookupTokenId,
+    lookupCreatorAddress,
+    lookupCollectionName,
+    lookupTokenName,
   ]);
   const readableAddress =
     originInfo &&
```

Comment:
Probably here also we can use `destructuring` like:

`const {chainId, assetAddress, ....} = originInfo || {}`

This also comes in handy for all `originInfo` properties used in the whole file.